production/helm/loki/templates/runtime-configmap.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
For the standard config, we support importing the config from a secret. I guess this makes sense for hiding credentials. I don't think it's necessary for the runtime config, but I'd like to know your opinion @trevorwhitney. Wdyt?

Reply:
Here's the conversation for the initial config #7358

Let's not put it into a secret for now. We could add the option to do so though.

Reply:
I agree with @jeschkies, keep a `ConfigMap` for now, and I'd like to see us implement a configuration as talked about in #7358 to specify `ConfigMap` vs `Secret` for both this and the regular config at a later point.